scripts/ci-checks-v2.sh

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env bash
+set -euo pipefail
+
+crates_to_check=\
+'api_models
+diesel_models
+hyperswitch_domain_models
+storage_impl'
+
+v2_feature_set='v2,merchant_account_v2,payment_v2,customer_v2,routing_v2,business_profile_v2'
+
+PACKAGES_CHECKED=()
+PACKAGES_SKIPPED=()
```

Comment:
Let's name all these variables which are only used within the script and not being exported in `snake_case` instead of `SCREAMING_SNAKE_CASE`:

```suggestion
packages_checked=()
packages_skipped=()
```

Same applies to `COMMON_COMMAND` and `CRATES_TO_INCLUDE`.